src/main/java/com/uid2/optout/util/Tuple.java

```diff
@@ -0,0 +1,31 @@
+package com.uid2.optout.util;
+
+public class Tuple {
```

Comment:
There already is a class like this, called Pair

Reply:
As discussed decided to keep Tuple